frontend/src/App.js

```diff
@@ -83,7 +80,7 @@ function App() {
           <Route path="/home" element={<Home />}></Route>
           <Route path="/signin" element={<Signin />}></Route>
           <Route path="/signup" element={<Signup />}></Route>
-          <Route path="/Buyer" element={<Buyer/>}></Route>
+          <Route path="/Buyer" element={<Buyer />}></Route>
```

Comment:
```suggestion
          <Route path="/buyer" element={<Buyer />}></Route>
```

Shall we stick with a single convention?